eth/backend.go

```diff
@@ -63,6 +63,9 @@ var (
 )
 
 type Config struct {
+	ChainConfig *core.ChainConfig // chain configuration
+
+	Testnet   bool   // Indicates whether we're serving the testnet
```

Comment:
Just a note so it's not forgotten, as discussed this should be dropped in favor of a `DefaultChainConfig`